app/services/core/persistent-stateful-service.ts

```diff
@@ -22,7 +22,7 @@ export abstract class PersistentStatefulService<TState extends object> extends S
   static get initialState() {
     const persisted = JSON.parse(localStorage.getItem(this.localStorageKey) as string) || {};
 
-    return merge({}, this.defaultState, persisted);
+    return this.filter(merge({}, this.defaultState, persisted));
```

Comment:
does `this` need to be changed to `(this.constructor as typeof PersistentStatefulService)` here as well

Reply:
Nope because `initialState` is static, so in this getter `this` already refers to the constructor.

Reply:
oh nice didn't catch that